docs/tutorials/executor_forces_tutorial.rst

```diff
@@ -134,48 +175,56 @@ This may take some additional browsing of the docs to complete.
 
 Write an alternative Calling Script similar to above, but with the following differences:
 
- 1. Add an additional worker directory so workers operate in ``/scratch/ensemble`` instead of the default current working directory.
+ 1. Set :ref:`libEnsemble's logger<logger_config>` to print debug messages.
  2. Override the MPIExecutor's detected MPI runner with ``"openmpi"``.
- 3. Set :ref:`libEnsemble's logger<logger_config>` to print debug messages.
- 4. Use the :meth:`save_libE_output()<tools.save_libE_output>` function to save the History array and ``persis_info`` to files after libEnsemble completes.
+ 3. Tell the allocation function to return results to the generator asychronously.
+ 4. Use the ensemble function :meth:`save_output()<ensemble.Ensemble.save_output>` to save the History array and ``persis_info`` to files after libEnsemble completes.
```

Comment:
Such paths need to be full, I think: `libensemble.ensemble.Ensemble.save_output`